### ISSUE TITLE
No longer able to create attachments for FSCloud profile

### ISSUE THREAD

**Issue by ocofaigh** [terraform-ibm-modules/terraform-ibm-scc-da#177]
The SCC DA is now failing with:

```
 2024/08/27 13:31:42 Terraform apply | Error: CreateAttachmentWithContext failed Necessary attachment parameters are not available to create or update attachment.
 2024/08/27 13:31:42 Terraform apply | {
 2024/08/27 13:31:42 Terraform apply |     "StatusCode": 400,
 2024/08/27 13:31:42 Terraform apply |     "Headers": {
 2024/08/27 13:31:42 Terraform apply |         "Cache-Control": [
 2024/08/27 13:31:42 Terraform apply |             "no-store"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "Cf-Cache-Status": [
 2024/08/27 13:31:42 Terraform apply |             "DYNAMIC"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "Cf-Ray": [
 2024/08/27 13:31:42 Terraform apply |             "8b9c6c3ec9716c04-DFW"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "Content-Length": [
 2024/08/27 13:31:42 Terraform apply |             "176"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "Content-Type": [
 2024/08/27 13:31:42 Terraform apply |             "application/json; charset=utf-8"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "Date": [
 2024/08/27 13:31:42 Terraform apply |             "Tue, 27 Aug 2024 13:31:42 GMT"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "Server": [
 2024/08/27 13:31:42 Terraform apply |             "cloudflare"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "Strict-Transport-Security": [
 2024/08/27 13:31:42 Terraform apply |             "max-age=31536000; includeSubDomains"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "Transaction-Id": [
 2024/08/27 13:31:42 Terraform apply |             "fba981a1-6299-4398-a7e4-71f17a07b3ef"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "X-Content-Type-Options": [
 2024/08/27 13:31:42 Terraform apply |             "nosniff"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "X-Correlation-Id": [
 2024/08/27 13:31:42 Terraform apply |             "4ad4e96e-6ff4-449f-9fc5-6e2753ed05c0"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "X-Envoy-Upstream-Service-Time": [
 2024/08/27 13:31:42 Terraform apply |             "6279"
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "X-Request-Id": [
 2024/08/27 13:31:42 Terraform apply |             "8972dd58-b752-4c30-a434-c0399d4b0544"
 2024/08/27 13:31:42 Terraform apply |         ]
 2024/08/27 13:31:42 Terraform apply |     },
 2024/08/27 13:31:42 Terraform apply |     "Result": {
 2024/08/27 13:31:42 Terraform apply |         "errors": [
 2024/08/27 13:31:42 Terraform apply |             {
 2024/08/27 13:31:42 Terraform apply |                 "code": "bad_request",
 2024/08/27 13:31:42 Terraform apply |                 "message": "Necessary attachment parameters are not available to create or update attachment."
 2024/08/27 13:31:42 Terraform apply |             }
 2024/08/27 13:31:42 Terraform apply |         ],
 2024/08/27 13:31:42 Terraform apply |         "trace": "4ad4e96e-6ff4-449f-9fc5-6e2753ed05c0"
 2024/08/27 13:31:42 Terraform apply |     },
 2024/08/27 13:31:42 Terraform apply |     "RawResult": null
 2024/08/27 13:31:42 Terraform apply | }
 2024/08/27 13:31:42 Terraform apply | 
 2024/08/27 13:31:42 Terraform apply | 
 2024/08/27 13:31:42 Terraform apply |   with module.create_profile_attachment["IBM Cloud Framework for Financial Services"].ibm_scc_profile_attachment.scc_profile_attachment,
 2024/08/27 13:31:42 Terraform apply |   on .terraform/modules/create_profile_attachment/modules/attachment/main.tf line 43, in resource "ibm_scc_profile_attachment" "scc_profile_attachment":
 2024/08/27 13:31:42 Terraform apply |   43: resource "ibm_scc_profile_attachment" "scc_profile_attachment" {
 2024/08/27 13:31:42 Terraform apply | 
```

This correlates to with the depreciation of `IBM Cloud Framework for Financial Services` version `1.6.0`. However version `1.7.0` is out now, so my understanding is our tests should be pulling the latest. Why are we getting this error?

**Comment by jor2** [comment]
https://github.com/IBM-Cloud/terraform-provider-ibm/issues/5586

**Comment by ocofaigh** [comment]
Fixed in 1.69.0. PR open https://github.com/terraform-ibm-modules/terraform-ibm-scc-da/pull/182
